packages/scope-manager/tools/generate-lib.ts

```diff
@@ -270,6 +280,17 @@ function main(): void {
 
   fs.writeFileSync(TYPES_FILE, formattedLibUnionCode);
   console.log('Wrote Lib union type file');
+
+  const lint = new ESLint({
+    fix: true,
+  });
+  const results = await lint.lintFiles(filesWritten);
+  await ESLint.outputFixes(results);
+  console.log('Autofixed lint errors');
 }
 
-main();
+main().catch(e => {
+  console.error(e);
+  // eslint-disable-next-line no-process-exit
+  process.exit(1);
+});
```

Comment:
What's going on here? Is it related to #7534?

Reply:
Nah so I ran the generator and noticed that it changed all of the lib files and caused a bunch of lint errors due to unsorted imports and such.

I figured that instead of manually running the linter afterwards was dumb cos we can automate it :)